### ISSUE TITLE
lack of precision over which version of WCAG we recommend meeting in Use plain English 

### ISSUE THREAD

**Issue by ElenaFdR** [publiccodenet/standard#412]
The hyperlink is to 2.1, but we don't say that.

**Comment by ericherman** [comment]
Section 3.1.5, to which we link, is the same in WCAG 2.0 and WCAG 2.1 and presumably will not meaningfully change in future version of WCAG 2.x, thus additional precision is not useful, or required.

Also, we do not wish to suggest that if 2.2 is released that you must stick to the 2.1 version.
